src/components/flat-table/flat-table.component.js

```diff
@@ -21,7 +25,16 @@ FlatTable.propTypes = {
   /** FlatTableHead and FlatTableBody */
   children: PropTypes.node.isRequired,
   /** If true, the header does not scroll with the content */
-  hasStickyHead: PropTypes.bool
+  hasStickyHead: PropTypes.bool,
+  /** Override `TableHeader` background color */
+  styleOverride: PropTypes.object,
+  /** Setup one of theme colors to the Table */
+  colorTheme: PropTypes.string
```

Comment:
Could this not be `oneOf(['transparent', 'white', 'dark', 'light'])` or you could use the `OptionsHelper.flatTableThemes` and then ensure the options are listed in the comment above so they display in the prop table in storybook

Reply:
fixed in https://github.com/Sage/carbon/pull/2866/commits/840d7cd4461c614101f121bf05a818cb440074d9